rholang/examples/linking/v0.2/tests/MakeBrandPairTest.rho

```diff
@@ -0,0 +1,46 @@
+import MakeBrandPair, TestSet in {
+  MakeBrandPair!("bp1") | MakeBrandPair!("bp2") |
+  contract @"BrandPairTest"(method, @input, return) = { method!(input, *return) } |
+  for(@r1 <- @"bp1"; @r2 <- @"bp2") {
+    match [r1, r2] {
+      [[seal, unseal], [otherSeal, otherUnseal]] => {
+        //seal composed with unseal == identity (modulo that it is now a single element list)
+        contract @"identity"(@input, return) = {
+          new sealed in {
+            @{seal}!(input, *sealed) | for(@s <- sealed) {
+              @{unseal}!(s, *return)
```

Comment:
It is interesting that we are forced to make return into a process even though it is only ever used as a channel throughout. It feels like a language smell.

Reply:
You always send processes and receive names. My style choice has been to tag things that are meant to be used as processes with an `@` in the contract definition and leave things that are supposed to be used as names alone. This serves as a sort of weak type indication, but yeah means that you need to cast names into processes sometimes when passing them on to another contract.

Reply:
@birchmd Maybe it would be better if we explicitly quoted before we sent because that is implicitly what is happening behind the scenes anyways. And I feel like it is more natural than unquoting a channel just to have it requoted.

Reply:
Or maybe specifically, if we have a GPrivate then we don't have to explicitly unquote.

Reply:
You mean make the send/receive symmetric so that you both send and receive names? Then you could send `return` directly and would need to explicitly quote `s`. I would be ok with that.